administrator/components/com_media/tmpl/file/default.php

```diff
@@ -58,10 +58,13 @@
 $this->useCoreUI = true;
 ?>
 <?php if ($tmpl === 'component') : ?>
-<div class="subhead noshadow mb-3">
-    <?php echo $this->getDocument()->getToolbar('toolbar')->render(); ?>
-</div>
+    <div class="subhead noshadow mb-3">
+        <?php echo $this->getDocument()->getToolbar('toolbar')->render(); ?>
+    </div>
```

Comment:
```suggestion
<div class="subhead noshadow mb-3">
    <?php echo $this->getDocument()->getToolbar('toolbar')->render(); ?>
</div>
```
Please revert the additional indentation here.